evap/evaluation/templates/comment_visibility_buttons.html

```diff
@@ -1,12 +1,12 @@
 <div class="btn-group btn-group-toggle" data-toggle="buttons">
     {% for choice in form.comment_visibility %}
-        {% if choice.data.value == 'ALL' %}
-            {% trans 'Will see all comments to general questions and about all non-responsible contributors.' as tooltip %}
-        {% elif choice.data.value == 'GENERAL' %}
+        {% if choice.data.value == 'GENERAL' %}
             {% trans 'Will see own and general comments.' as tooltip %}
+            {% trans 'Own and general' as label %}
         {% elif choice.data.value == 'OWN' %}
             {% trans 'Default value. Will only see own comments.' as tooltip %}
+            {% trans 'Own' as label %}
```

Comment:
having "own" and "general" on the buttons doesn't really help the users to understand what's happening. especially since we're now only having two options it must be made clear that the second option includes the first and something in addition.

Reply:
what if we put the more verbose string into the `choices` tuple? where do we need the short string?

Reply:
We can put it there, the short version is never used.